administrator/language/de-DE/plg_task_privacyconsent.ini

```
@@ -0,0 +1,20 @@
+; Joomla! German Translation
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2023 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_TASK_PRIVACYCONSENT="Aufgabe – Datenschutz Zustimmung"
+PLG_TASK_PRIVACYCONSENT_CONSENTEXPIRATIONDAYS_DESC="Anzahl der Tage, nach denen die Datenschutz Zustimmung abläuft."
+PLG_TASK_PRIVACYCONSENT_CONSENTEXPIRATIONDAYS_LABEL="Zustimmungsdauer"
+PLG_TASK_PRIVACYCONSENT_INVALIDATE_TITLE ="Ablauf der Datenschutz Zustimmung"
+PLG_TASK_PRIVACYCONSENT_INVALIDATE_DESC ="Verwaltet das Auslaufen von Datenschutzzustimmungen"
```

```suggestion
PLG_TASK_PRIVACYCONSENT_INVALIDATE_TITLE="Ablauf der Datenschutz Zustimmung"
PLG_TASK_PRIVACYCONSENT_INVALIDATE_DESC="Verwaltet das Auslaufen von Datenschutzzustimmungen"
```
siehe auch https://github.com/joomla/joomla-cms/pull/41724